File: medplum-java/medplum-model/src/main/java/com/fhir/TestScript_Teardown.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableTestScript_Teardown.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface TestScript_Teardown {
  Optional<List<Extension>> extension();

  List<TestScript_Action2> action();

  Optional<String> id();

  Optional<List<Extension>> modifierExtension();

  static ImmutableTestScript_Teardown.Builder builder() {
    return ImmutableTestScript_Teardown.builder();
  }
}

File: medplum-typescript/src/model/fhir/TestScript_Teardown.ts
import { Extension } from "./Extension";
import { TestScript_Action2 } from "./TestScript_Action2";

export interface TestScript_Teardown {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    action: TestScript_Action2[];
}
